app/controllers/posts_controller.rb

```diff
@@ -10,5 +10,23 @@ def show
     @post = Post.find(
       params[:id]
     )
+    @comment = Comment.new
+    @like = Like.new
+  end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = Post.new(params.require(:post).permit(:title, :text))
+    author = User.find(params[:user_id])
+    @post.author = author
```

Comment:
- [ ] It would be better if we create the posts on behalf of the current_user as we did to the other controllers. Let us kindly use the method created in the ApplicationController
![Screenshot from 2023-09-28 12-13-09](https://github.com/K0ppai/Blog_Web/assets/46388113/9e3a8cb7-17f1-4523-ab14-915a859af2db)